content/md/productivity/get-started-with-the-rules-engine.md

```diff
@@ -832,38 +861,68 @@ We expect **a number** as `value`.
 
 The possible operators for the `Date` attribute type are:  
 - <
-- ">"
+- '>'
 - =
 - !=
 - BETWEEN
 - NOT BETWEEN
 - EMPTY
 - NOT EMPTY
 
-The expected date format is: **yyyy-mm-dd H:i:s**
+::: info
+The format of the date is:
+- "yyyy-mm-dd"
+- "now"
+- "\<relative date format\>" (see [below](#focus-on-the-relative-date-format-2))
 
-:::info
 If the operator is EMPTY or NOT EMPTY, the value element will be ignored.
```

Comment:
maybe add a list of operators supported with relative dates (same as created/updated, plus ">=" and "<=")